label_studio_ml/examples/gliner/README.md

```diff
@@ -0,0 +1,59 @@
+The GLiNER model is a BERT family model for generalist NER. We download the model from HuggingFace, but the original
```

Comment:
@caitlinwheeless can you review the README and propose the head for this file to match the gallery?

Reply:
Added